Application/EdFi.Ods.SandboxAdmin/Views/Application/Index.cshtml

```diff
@@ -0,0 +1,208 @@
+﻿<!--
+  SPDX-License-Identifier: Apache-2.0
+  Licensed to the Ed-Fi Alliance under one or more agreements.
+  The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+  See the LICENSE and NOTICES files in the project root for more information.
+-->
+
+@{
+    ViewBag.Title = "Sandbox Administration Portal";
+    Layout = "~/Views/Shared/Layout.cshtml";
+}
+
+<script src="~/js/application.js"></script>
+
+
+<div class="row">
+    <div class="jumbotron">
+        <h3>Sandbox Administration Portal for the Ed-Fi ODS API</h3>
+        <p class="lead">
+            This portal enables you to manage sandboxes for the Ed-Fi ODS API, including the keys and secrets for accessing these sandboxes.
+        </p>
+    </div>
+</div>
+
+<div class="row">
+    <div class="alert alert-danger" data-bind="visible: error">
+        <strong>Error:</strong> <span data-bind="text: error"></span>
+    </div>
+</div>
+
+<div class="container-fluid">
+    <div class="row">
+        <div class="col-md-12">
+            <div class="float-md-right">
+                <button id="add-application" class="btn btn-primary" data-bind="enable: canAddMoreApplications, click: addApplicationClicked">Add Application</button>
+            </div>
+    <fieldset>
+        <legend>Existing Applications   </legend>
+        <table class="table table-hover" id="clients-table">
+            <thead>
+                <tr>
+                    <th></th>
+                    <th>Application Name </th>
+                    <th>EducationOrganizationId</th>
+                    <th>Vendor Name</th>
+                    <th></th>
+                </tr>
+            </thead>
+            <tbody data-bind="foreach: applications">
+                <tr data-bind="attr: { 'data-applicationId': $data.Id, 'data-applicationname': $data.Name }">
+                    <td>
+                        <div class="btn-group">
+                            <div class="dropdown">
+                                <button class="btn btn-warning dropdown-toggle" id="sandbox-menu" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                                    <i class="fa fa-cog" aria-hidden="true"></i>
+                                    <span class="caret"></span>
+                                </button>
+                                <div class="dropdown-menu" aria-labelledby="sandbox-menu">
+                                    <a href="javascript:void(0)" data-bind="click: $parent.updateApplicationClicked" class="editApplication dropdown-item">
+                                        Edit
```

Comment:
This `Edit` button doesn't seem to do anything (I see a js error in the developer console whenever I click it, that might be the reason), can you take a look?

![image](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation/assets/158314644/1640f29a-eb5f-4780-b9f4-c369efe0f31a)

Reply:
No longer an issue since the Edit functionality got removed.